CHANGELOG.rst

```diff
@@ -2,30 +2,52 @@
 Changelog
 *********
 
-0.1.0 -- 2019-02-05
-===================
-* Initial public release
+1.0.1 -- 2019-09-27
+=================== 
+* Merge pull request #376 from lucasmt/dependency-graph
```

Comment:
For the previous releases we added a summary of changes and did not detail  the Pull Requests we merged, as it would create a long list of changes.

Reply:
Reduced to just PRs